frontend/src/__tests__/components/flex.test.js

```diff
@@ -0,0 +1,63 @@
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import Flex from '../../../components/Flex/Flex';
+import Colors from '../../../components/colors';
+
+describe('Flex Component Tests', () => {
+  it('should render a default Flex component', () => {
+    render(<Flex />);
+    const flexComponent = screen.getByTestId('flex-1');
+    expect(flexComponent).toBeInTheDocument();
+    expect(flexComponent).toHaveStyle(
+      `background-color : ${Colors.Mono[0]}`,
```

Comment:
I did this to allow for the color theme changing in the future. If the color swatch (spelling?) is changed/moved then it wont affect the test as the least color in the chain will remain in position 0 ( and the greatest in the last position). 

If possible could i keep the implementation as it is? I ask becauase if static testing is not a requirement of the capstone i believe it will be some work without any intrinsic benefit. If i were to change the colors in the testing, would i also not need to change it in the components themselves?

Reply:
If the color theme changes in the future, we should have some sort of warning that let's us know we need to verify the behavior of components that it affects (i.e. update the color it's checking for in the test). Additionally, after a certain point (while this example is rather timid) dynamic testing can become rather complicated. It's best to just stick to the standard that we have across DU as that is the way you will be testing things on the real project.

Tests are not just written so we can forget about them later, they are there to let us know when things aren't acting the way they are supposed to. If that component originally had a white background, and we change our color theme to black - we need to do the due diligence as a developer to go check that component out and update the tests and verify functionality.